app/src/main/java/com/hewgill/android/nzsldict/SignVideoFragment.java

```diff
@@ -83,9 +127,13 @@ public void onPrepared(MediaPlayer mp) {
             }
         });
 
+        // Start loading video if network is available
+        if (isNetworkAvailable()) updateViewForConnectivityStatus();
```

Comment:
You check isNetworkAvailable again in the `updateViewForConnectivity`.. so it's not needed here.